app/src/main/java/com/example/githubapp/repos/ReposFragment.kt

```diff
@@ -60,4 +64,13 @@ class ReposFragment : Fragment() {
         binding.list.adapter = adapter
     }
 
+    private fun search(query: String) {
+        searchJob?.cancel()
```

Comment:
In MVVM the View layer should be kept as simple as possible. All presentation logic should be located in a ViewModel. 
In this case, the fragment should observe the items list as it observed before, and send the search events to the ViewModel (call `getRepos()` method). The job cancelation should be handled by the ViewModel.